src/Sulu/Bundle/LocationBundle/Resources/js/index.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import {fieldRegistry} from 'sulu-admin-bundle/containers';
+import L from 'leaflet';
```

Comment:
Isn't a one-letter import weird?

Reply:
absolutely 😂 but leaflet uses this style throughout its documentation. we can still change it, dont have any opinion here. https://leafletjs.com/examples/quick-start/